ansible.cfg

```diff
@@ -1,6 +1,6 @@
 [defaults]
 inventory = ./inventory.yml
-vault_password_file = ./vault.sh
+# vault_password_file = ./vault.sh
```

Comment:
TODO: somehow the config value still overrides the vault-password-file passed with CLI